web-api-shared/src/elastic.rs

```diff
@@ -65,6 +68,11 @@ impl Default for Config {
             password: String::from("changeme").into(),
             index_name: "test_index".into(),
             timeout: Duration::from_secs(2),
+            retry_policy: ExponentialJitterRetryPolicyConfig {
+                max_retries: 2,
+                step_size: Duration::from_millis(150),
+                max_backoff: Duration::from_millis(1000),
+            },
```

Comment:
No idea what good values are.

In a app we probably would use 1s or similar for step size, but because it's internal requests going with something lower should be preferable.